src/Admin/Vue/PaymentMethods.php

```diff
@@ -142,10 +148,10 @@ public function one_click_option( $active = null ) {
 				]
 			]
 		];
-		if (isset(get_option( 'woocommerce_payplug_settings', [] )['oneclick'])) {
-			if (get_option( 'woocommerce_payplug_settings', [] )['oneclick'] != 'no') {
+		if (isset($this->options['oneclick'])) {
```

Comment:
why not do this add oneclick logic before and add the position on the array when you're declaring it above? (more efficient)

also if oneclick is not isset, the position should exist and be false right?

Reply:
done